index.html

```diff
@@ -0,0 +1,152 @@
+        </div>
+        <div class="box">
+          <i class="fa-solid fa-graduation-cap"></i>
+          <span>2019</span>
+          <h3>university student</h3>
+          <p>i started my high education at kabul university faculty of pharmacy, it was the best journey of my life which i started, i gone head which hight grade and could took position of highbrow</p>
+        </div>
+      </div>
+      <div class="box-container">
+        <div class="box">
+          <i class="fa-solid fa-graduation-cap"></i>
+          <span>2020</span>
+          <h3>university student</h3>
+          <p>in these years beside that i was student i started to learn english as high level and took preparation for TOFEL test and computer course to improve self skills.</p>
+        </div>
+        <div class="box">
+          <i class="fa-solid fa-graduation-cap"></i>
+          <span>2024</span>
+          <h3>web development student</h3>
+          <p>this year is more usable for me because i have started learning web development and want to improve my skills in this part this is the only way to llearn from long distance.</p>
+        </div>
+      </div>
+    </section>
+    <section class="portfolio" id="portfolio">
+      <h1 class="heading"> My <span>portfolio</span></h1>
+      <div class="box-container">
+        <div class="box">
+          <div>
+            <img src="./assets/landing page.jpg" alt="">
+          </div><br><br>
+          <h3>landing page</h3>
+          <button><a href="https://github.com/sodabarauf/product-landing-page">source code</a></button><br>
+          <button><a href=" https://sodabarauf.github.io/product-landing-page/">live demo</a></button>
+        </div>
+        <div class="box">
+          <div>
+            <img src="./assets/tribute page.jpg" alt="">
+          </div><br><br>
+          <h3>tribute page</h3>
+          <button><a href="https://github.com/sodabarauf/tribute">source code</a></button><br>
+          <button><a href="">live demo</a></button>
+        </div>
+        <div class="box">
+          <div>
+            <img src="./assets/documentation shot.jpg" alt="">
+          </div><br><br>
+            <h3>documentation page</h3>
+            <button><a href="https://github.com/sodabarauf/technical-documentation-page">source code</a></button><br>
+            <button><a href=" https://sodabarauf.github.io/technical-documentation-page/">live demo</a></button>
+        </div>
+        <div class="box">
+          <div>
+            <img src="./assets/capture of portfolio.jpg" alt="">
+          </div><br><br>
+            <h3>portfolio</h3>
+            <button><a href="https://github.com/sodabarauf/portfolio">source code</a></button><br>
+            <button><a href="https://sodabarauf.github.io/portfolio/" id="demolink">live demo</a></button>
+        </div>
```

Comment:
- Remove this project and instead add your survey form project here

![Screenshot 2024-06-08 145803](https://github.com/sodabarauf/main-portfolio/assets/96838030/7c529d7e-1dc0-45fe-88d3-0908e9060d9e)